pkg/client/src/app/pages/applications/analysis-wizard/components/upload-binary.tsx

```diff
@@ -40,11 +47,17 @@ export const UploadBinary: React.FunctionComponent<IUploadBinary> = ({
   }, []);
 
   const completedUpload = (response: any) => {
+    dispatch(
+      alertActions.addSuccess(`Task ${taskId}`, `Uploading binary file.`)
```

Comment:
Ah I wonder if "Binary uploaded successfully." would be a better fit for the text here?